src/main/java/mate/academy/bookstore/repository/impl/BookRepositoryImpl.java

```diff
@@ -46,7 +48,17 @@ public List<Book> findAll() {
         try (Session session = sessionFactory.openSession()) {
             return session.createQuery("FROM Book", Book.class).getResultList();
         } catch (Exception e) {
-            throw new DataProcessingException("Can't find all books", e);
+            throw new EntityNotFoundException("Can't find all books", e);
+        }
```

Comment:
If an exception occurs while retrieving the books from DB, but there are some books there, will it be correct to tell the user that you can't find the books?